src/Course.java

```diff
@@ -0,0 +1,12 @@
+public class Course {
+    String courseName;
```

Comment:
apply [encapsulation](https://www.tutorialspoint.com/java/java_encapsulation.htm)

make class properties `private` and provide `public` getter/setter methods

i.e.
``  
private String Name
public String getName() {
   return name;
}

public void setName(String name) {
    this.name = name;
}
``  

you can use intellij shorcut to generate such getter/setter method with - `Alt+Ins`